### ISSUE TITLE
Prediction on DHF1K

### ISSUE THREAD

**Issue by research-boy** [WZq975/STSANet#1]
I am new to saliency problem, i want to generate the saliency map similar to the labels , i tried normalizing and multiplying with 255.
this is what the output looks like. Using your weight : STSANet_DHF1K.pth. 
![image](https://github.com/WZq975/STSANet/assets/46345379/5e386876-b6d4-4bc4-8e5c-882e83c88e53)
It would be great if you can give some guidance regrading this. Thanks

**Comment by WZq975** [comment]
Hi,

You can refer to the following function to save the prediction.
```
def save_prediction(prediction, save_name, save_size=(384, 224)):
    prediction = prediction.cpu().squeeze().clone().numpy()
    prediction = cv2.resize(prediction, save_size)
    prediction_norm = (prediction - np.min(prediction)) / (
            (np.max(prediction) - np.min(prediction) + 0.00001) * 1.0)
    prediction_norm = np.round(prediction_norm * 255 + 0.5)
    prediction_norm = np.asarray(prediction_norm, dtype=np.uint8)
    cv2.imwrite(save_name, prediction_norm)
```
Also, the keys in the weight "STSANet_DHF1K.pth" have the prefix "module.", so if you don't use "torch.nn.DataParallel", you need to remove the prefix when loading the weight like:
```
weights = torch.load('STSANet_DHF1K.pth')
weights = {k.split('module.')[1]: v for k, v in weights.items()}
```

Best,
